webui/src/pages/repositories/repository/index.jsx

```diff
@@ -19,9 +20,12 @@ const RepositoryPage = () => {
             <Route exact path="/">
                 <Redirect to="/repositories"/>
             </Route>
-            <Route path="/repositories/:repoId/objects">
+            <Route exact path="/repositories/:repoId/objects">
                 <RepositoryObjectsPage/>
             </Route>
+            <Route path="/repositories/:repoId/objects/:objectName">
```

Comment:
Doesn't work with react devserver [when the filename contains dots](https://stackoverflow.com/questions/36923466/dot-and-hyphen-disallowed-react-router-url-parameters).

Reply:
The webpack dev server is different than the Vite dev server.
Unfortunately, the Vite dev server doesn't have such a convenient toggle.
After much research, turns out Vite uses [sirv](https://www.npmjs.com/package/sirv) for static assets and there's no way to configure it externally.
As an alternative, we could implement a custom middleware, since Vite exposes the raw express app, so you can add middlewares on. However, this might create other issues with resolving the root `index.html` and such.
IMHO, for now I think we should keep it as a known issue.